src/AppBundle/Entity/Client.php

```diff
@@ -288,13 +303,13 @@ public function getFullname()
 
     public function setCourtDateWithoutTime($courtDate = null)
     {
-        $this->courtDate = ($courtDate instanceof \DateTime) ?
-                new \DateTime($courtDate->format('Y-m-d')) : null;
+        $this->courtDate = ($courtDate instanceof DateTime) ?
+                new DateTime($courtDate->format('Y-m-d')) : null;
     }
 
     public function isValidCourtDate(ExecutionContextInterface $context)
     {
-        $today = new \DateTime();
+        $today = new DateTime();
```

Comment:
Can you check this is the correct object? I always used \DateTime but could be wrong